controller/controller_server/common_config.py

```diff
@@ -0,0 +1,23 @@
+import os.path
+
+import yaml
+
+
+class CommonConfig:

```

Comment:
```suggestion
class Config:

```
if we ever add more configurations, we could always undo this suggestion, to distinguish

Reply:
done

Reply:
there is [other configs](https://github.com/IBM/ibm-block-csi-driver/blob/1fbec0ca30ceb6fc44bcc28a5ba945246eec890e/controller/controller_server/csi_controller_server.py#L6).. so I changed where [I could](https://github.com/IBM/ibm-block-csi-driver/blob/1fbec0ca30ceb6fc44bcc28a5ba945246eec890e/controller/controller_server/controller_server_manager.py#L11)..

Reply:
oh, missed it. thanks